### ISSUE TITLE
Cannot find the option of jump to next tab in input gesture dialogue

### ISSUE THREAD

**Issue by LittleStar-VIP** [nvaccess/nvda#15711]
### Steps to reproduce:
1. make sure the focus is in browse mode, e.g. focus on the document of a webpage
2. use nvda +n open nvda menue
3. preferences>input dialogue
4. under browse mode cannot find the option of jump to next tab
since tab is start with t, so I double check with jump to next txxx, I could only find jump to next table, and before that and after that option, there is no jump to next tab
I have also look through the whole list of jump to next xxx, and no jump to next tab is found.

### Expected behavior:
I should be able to find this option 'jump to next tab'

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2023.3
#### Windows version:
windows 10 home ed 22h2
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.

I have repeated checking whether there is the option of jump to next tab in nvda 2023.3 rc1, rc2, beta4

And I can only find the option of jump to next tab in beta4 when I set the nvda language to english. Since my default nvda language is zh-tw Traditional Chinese Taiwan, if I switch back to zh-tw, the option of jump to next tab cannot be found in beta4.
for nvda 2023.3 rc1, rc2, the option cannot be found neither in english nor in zh-tw

#### If NVDA add-ons are disabled, is your problem still occurring?
I have tried restart with addons disabled and the problem is still here.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
not tried

**Comment by lukaszgo1** [comment]
This is quite strange. For me with NVDA 2023.3 running from the launcher, and language set to English 'moves to the next tab' is present in the input gestures dialog. The script is called 'moves to the next tab', so perhaps that is why you cannot find it by searching for 'jump'.

**Comment by Adriani90** [comment]
I cannot reproduce this issue with german, english or romanian language. The jump to next tab is the item number 37, of 76 and the jump to previous tab is item 52 of 76. Did you try to uninstall NVDA, delete the user config and reinstall NVDA?

I guess it might be something wrong with your NVDA installation.

**Comment by Adriani90** [comment]
Also in which browser are you having this issue? I tested in MS Word, Firefox, Chrome, MS Edge and Opera browser and in all scenarios I can find the tab in the input gestures list under browse mode.

**Comment by britechguy** [comment]
I, too, have not as yet been able to find this option in Browse mode options.

I'm looking at the GitHub page for this issue, in Vivaldi, using NVDA 2023.3, freshly installed last night, and my list of "moves to next" goes from separator, to table, to unvisited link.  When in an MS Word document, the only "moves to next/previous" option is for articles.

I have just restarted NVDA with add-ons disabled and I do get the "moves to the next/previous tab" option, so there appears to be interference from an add-on or add-ons.  Having disabled half of my add-ons, the "moves to the next/previous tab" only reappears when I reenable Browser Nav, so I will report this issue to Tony Malykh via his GitHub Page.

So it's on record, here are the Add-Ons I have installed:
Add-ons documentation
BluetoothAudio
BrowserNav
Easy Table Navigator
Golden Cursor
NoBeepsSpeechMode
ObjPad
Office Desk
Place Markers
Remote Support
Speech Logger
TextNav
Thunderbird+G5 (TB >= 115)
Version Collector
Virtual Review

**Comment by seanbudd** [comment]
Closing as this is an issue with an add-on

**Comment by LittleStar-VIP** [comment]
@Adriani90 
I tried to create a clean portal of nvda 2023.3 and I was able to find the jump to next tab option.
So probably I have to uninstall the current nvda and install a new one?
But what might lead to this problem? nvda is user profile is corrupted?

**Comment by seanbudd** [comment]
@LittleStar-VIP - as noted by @britechguy the issue is with the "Browser Nav" add-on.

**Comment by LittleStar-VIP** [comment]
@seanbudd 
Oh thank you.
